### PR TITLE
DG: Manual testing for pay, owe and settle command 

### PR DESCRIPTION
**Pull request by teamotitan**
Previously, manual testing for pay, owe and settle command is not present in the DG

Let's
* include manual testing for these commands in our DG

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -614,3 +614,92 @@ testers are expected to do more *exploratory* testing.
...
+
+1. Using settle command from a filtered list.
+
+    1. Prerequisite: Find a student using the `find` command. There should be **at least 1 student** found.
+
+    1. Test case: `settle 1 amount/10`<br>
+       Expected: For 1st student in **filtered list**, assuming owed amount is more than 10, owed amount decreases by 10, while paid amount increases by 10. UGTeach displays a message that payment of 10.00 has been settled.
+
+    1. Test case: `settle 1 amount/-10`<br>
+       Expected: No changes. UGTeach displays error message that amount needs to be at most 2 decimal places and must be between
```
Comment (Feng1231) [inline docs/DeveloperGuide.md:700]: I don't see the need to elaborate all these error messages. Your line 682 and line 699 seems to test for the same thing but expected is different.
`Displays an error message` is enough.

**Review by Feng1231 — CHANGES_REQUESTED** [review]
Don't have to elaborate for the error messages. Will be very troublesome if we change the content in the error message.

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (Soumyadip-cmd) [reply docs/DeveloperGuide.md:700]: I think he forgot to change the expected for line 682. But I think can perhaps say 'displays error message notifying that amount is invalid' for where ever applicable

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -614,3 +614,92 @@ testers are expected to do more *exploratory* testing.
...
+
+1. Using pay command while all students are being shown. 
+   
+    1. Prerequisite: List all students using the `list` command. There should be **at least 1 student** listed. 
+
+    1. Test case: `pay 1 hr/1`<br>
+       Expected: Paid amount for 1st student increases by (1 hr * rate). UGTeach displays a message that 1st student paid (1 hr * rate).
+   
+    1. Test case: `pay 1 hr/-1`<br>
+       Expected: No changes. UGTeach displays error message asking for positive multiple of 0.5 when inputting hours.
```
Comment (Soumyadip-cmd) [inline docs/DeveloperGuide.md:627]: Similar to comment for settle

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -614,3 +614,92 @@ testers are expected to do more *exploratory* testing.
...
+
+1. Using pay command from a filtered list.
+
+    1. Prerequisite: Find a student using the `find` command. There should be **at least 1 student** found.
+
+    1. Test case: `pay 1 hr/1`<br>
+       Expected: Paid amount for 1st student in **filtered list** increases by (1 hr * rate). UGTeach displays a message that 1st student paid (1 hr * rate).
+
+    1. Test case: `pay 1 hr/-1`<br>
+       Expected: No changes. UGTeach displays error message asking for positive multiple of 0.5 when inputting hours.
```
Comment (Soumyadip-cmd) [inline docs/DeveloperGuide.md:640]: Similar to comment for settle

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -614,3 +614,92 @@ testers are expected to do more *exploratory* testing.
...
+
+1. Using owe command while all students are being shown.
+
+    1. Prerequisite: List all students using the `list` command. There should be **at least 1 student** listed.
+
+    1. Test case: `owe 1 hr/1`<br>
+       Expected: Owed amount for 1st student increases by (1 hr * rate). UGTeach displays a message that 1st student owed another (1 hr * rate).
+
+    1. Test case: `owe 1 hr/-1`<br>
+       Expected: No changes. UGTeach displays error message asking for positive multiple of 0.5 when inputting hours.
```
Comment (Soumyadip-cmd) [inline docs/DeveloperGuide.md:655]: Similar to comment for settle

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -614,3 +614,92 @@ testers are expected to do more *exploratory* testing.
...
+
+1. Using owe command from a filtered list.
+
+    1. Prerequisite: Find a student using the `find` command. There should be **at least 1 student** found.
+
+    1. Test case: `owe 1 hr/1`<br>
+       Expected: Owed amount for 1st student in **filtered list** increases by (1 hr * rate). UGTeach displays a message that 1st student owed another (1 hr * rate).
+
+    1. Test case: `owe 1 hr/-1`<br>
+       Expected: No changes. UGTeach displays error message asking for positive multiple of 0.5 when inputting hours.
```
Comment (Soumyadip-cmd) [inline docs/DeveloperGuide.md:668]: Similar to comment for settle

**Review by Soumyadip-cmd — COMMENTED** [review]
Can perhaps say error message notifying that number of hours provided is invalid

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (Feng1231) [reply docs/DeveloperGuide.md:700]: Sounds good

**Review by Feng1231 — APPROVED** [review]
Lgtm

File: docs/UserGuide.md [hunk 6]
```diff
@@ -61,7 +61,7 @@ Action     | Format, Examples
 **Delete** | `delete INDEX`<br> e.g., `delete 3`
 **Find**   | `find [n/KEYWORD [MORE_KEYWORDS]] [d/DAY [MORE_DAYS]]`<br> e.g., `find n/Alex d/Friday`
 **Pay**   | `pay INDEX hr/HOURS_PAID`<br> e.g., `pay 1 hr/2.5`
-**Owe**    | `owe INDEX hr/HOUR_OWED`<br> e.g., `owe 1 hr/1.5`
+ **Owe**    | `owe INDEX hr/HOUR_OWED`<br> e.g., `owe 1 hr/1.5`
```
Comment (Feng1231) [inline docs/UserGuide.md:64]: Is there a extra space here? Also, please help to change it to `HOURS_OWED`

**Review by Soumyadip-cmd — APPROVED** [review]
LGTM

**Review by Feng1231 — CHANGES_REQUESTED** [review]
Small rectification needed, thanks for updating

File: docs/UserGuide.md (hunk 6 above)
Comment (teamotitan) [reply docs/UserGuide.md:64]: Eh what the....thanks a alot

**Review by Feng1231 — APPROVED** [review]
LGTM

**Pull request merged**